test/prototype_transforms_kernel_infos.py

```diff
@@ -806,6 +816,43 @@ def sample_inputs_rotate_bounding_box():
         )
 
 
+def reference_inputs_rotate_bounding_box():
```

Comment:
@vfdev-5 this is new in this PR, right?

Reply:
yes, this is new to test rotation on bbox